### DISCUSSION TITLE
how to render emoji in a label?

### DISCUSSION THREAD

**Discussion by ITwrx** [can-lehmann/owlkettle#61] [Q&A] [answered]
I'm wanting to show a smiley face emoji in a label. Not seeing how to do that anywhere.

thanks

**Comment by can-lehmann** [comment, marked as answer]
If you use UTF-8 for encoding your source file, this should just work:

```nim
Label:
  text = "🙂"
```

**Comment by ITwrx** [comment]
> If you use UTF-8 for encoding your source file

i don't know about this^ any link to nim docs?

**Reply by can-lehmann** [reply]
UTF-8 is simply a text encoding. Your text editor probably already uses it anyways. I am not sure how nim handles different encodings, which is why I specified UTF-8. Does just embedding the emoji into the string not work for you?

**Comment by ITwrx** [comment]
It does now that i just pasted the emoji you posted. I forgot i'm supposed to just paste the final product, not some code for it.

thanks!
